crypto/rsa/rsa_mp.c

```diff
@@ -105,9 +105,6 @@ int ossl_rsa_multip_cap(int bits)
         cap = 3;
     else if (bits < 8192)
         cap = 4;
-
-    if (cap > RSA_MAX_PRIME_NUM)
-        cap = RSA_MAX_PRIME_NUM;
```

Comment:
Since `RSA_MAX_PRIME_NUM` is a constant defined elsewhere, and since it is used to size arrays, it would be best to keep its use in this function. I would suggest replacing `5` with `RSA_MAX_PRIME_NUM` at line 100, but then also consider what would happen if the value of `RSA_MAX_PRIME_NUM` is reduced in the future